src/modules/services_manager_target.rb

```diff
@@ -139,8 +140,15 @@ def inspect
 
     def save
       return true unless modified?
+
       log.info('Saving default target...')
-      Yast2::Systemd::Target.set_default(default_target)
+      unless Yast2::Systemd::Target.find(self.default_target)
```

Comment:
Please, don't use `unless` for multiliner `if`. Otherwise, rubocop complains with "Favor modifier unless usage when having a single-line body". Yes, maybe this package is not using rubocop at all, but even though we should follow the standard.